### PR TITLE
Restructure main text documents

### PR DESCRIPTION
**Pull request by aradi**
Make readme more informative, add table of successful builds to install instructions, change license formatting.

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -52,11 +52,49 @@ results, you will additionally need:
 * The Slater-Koster data used in the tests (see below)
 
 
+Successful builds
```
Comment (bhourahine) [inline INSTALL.rst:55]: Tested builds

File: INSTALL.rst [hunk 2]
```diff
@@ -52,11 +52,49 @@ results, you will additionally need:
...
+-----------------
+
+DFTB+ is regularly built and tested in following build environments:
```
Comment (bhourahine) [inline INSTALL.rst:58]: Table isn't clear that compilation without MPI is also possible. Suggest something like

DFTB+ is regularly built and tested for both serial and MPI  environments on the following architectures:

File: INSTALL.rst [hunk 3]
```diff
@@ -52,11 +52,49 @@ results, you will additionally need:
...
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | NAG Fortran 7.0      | MPICH 3.3   | OpenBlas 0.3.7   |     |
+| Linux         | GNU C 9.2            |             | ScaLAPACK 2.1    |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | GNU Fortran/C 8      | --          | LAPACK / BLAS    | [1] |
+| OS X          |                      |             |                  |     |
+|               |                      |             |                  |     |
++---------------+----------------------+-------------+------------------+-----+
+
+All builds use ARPACK-NG 3.7, ELSI 2.6.1 and PLUMED 2.5.
```
Comment (bhourahine) [inline INSTALL.rst:86]: All builds are also tested with the optional ARPACK-NG 3.7, ELSI 2.6.1 and PLUMED 2.5 libraries.

File: README.rst [hunk 4]
```diff
@@ -2,20 +2,136 @@
...
-method. The most recent features are described `here
-<https://doi.org/10.1063/1.5143190>`_. It can be either used as a
-standalone program or integrated into other software packages as a
-library.
+|lgpl badge|
 
-The project is hosted on `github <http://github.com/dftbplus/dftbplus>`_.
+DFTB+ is a software package for carrying out fast quantum mechanical atomistic
+calculations based on the Density Functional Tight Binding method. The most
+recent features are described in the `DFTB+ paper (open access)
```
Comment (bhourahine) [inline README.rst:9]: in the (open access) `DFTB+ paper <https://doi.org/10.1063/1.5143190>`_.

File: README.rst [hunk 5]
```diff
@@ -2,20 +2,136 @@
...
+* Methodological papers relevant to your calculations (e.g. excited states,
+  electron-transport, third order DFTB etc.). Those references can be found in
+  the `DFTB+ manual
+  <http://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/latest/manual.pdf>`_.
+
+
+Contributing
+============
+
+Welcome in the DFTB+ developer community!
```
Comment (bhourahine) [inline README.rst:112]: Not quite sure  what this is saying?

New features, bug fixes, tutorial examples and code testing is welcome in the DFTB+ developer community!

File: README.rst [hunk 6]
```diff
@@ -2,20 +2,136 @@
...
+
+The project is `hosted on github <http://github.com/dftbplus/dftbplus>`_.
```
Comment (bhourahine) [inline README.rst:114]: Might be a good idea to give the github.com/dftbplus address as the project, as this then includes the other components of the project, not just DFTB+.

File: INSTALL.rst (hunk 1 above)
Comment (aradi) [reply INSTALL.rst:55]: :heavy_check_mark:

**Comment by aradi** [comment]
Thanks! I've incorporated your suggetions.

**Comment by bhourahine** [comment]
Might also be  good to give some hints what to do if you are building for an unsupported environment (generic in sys/)

**Comment by aradi** [comment]
Good idea! I added a reference to that.

File: INSTALL.rst [hunk 7]
```diff
@@ -349,6 +394,6 @@ The customized config file is read by CMake before the compiler detection. If
 your config file contains toolchain dependent options, consider to define the
 ``DFTBPPLUS_TOOLCHAIN`` environment variable and query it in your config file.
 
-See this [CMake customization
-file](https://gist.github.com/aradi/39ab88acfbacc3b2f44d1e41e4da15e7) for a
+See this `CMake customization file
+<https://gist.github.com/aradi/39ab88acfbacc3b2f44d1e41e4da15e7>`_ for a
```
Comment (bhourahine) [inline INSTALL.rst:398]: Probably should be self-contained within repository

**Pull request merged**